bindgen-cli/options.rs

```diff
@@ -289,112 +306,128 @@ where
                 .default_value(DEFAULT_ANON_FIELDS_PREFIX),
             Arg::new("time-phases")
                 .long("time-phases")
-                .help("Time the different bindgen phases and print to stderr"),
+                .help("Time the different bindgen phases and print to stderr")
+                .action(ArgAction::SetTrue),
             // All positional arguments after the end of options marker, `--`
-            Arg::new("clang-args").last(true).multiple_occurrences(true),
+            Arg::new("clang-args").last(true).action(ArgAction::Append),
             Arg::new("emit-clang-ast")
                 .long("emit-clang-ast")
-                .help("Output the Clang AST for debugging purposes."),
+                .help("Output the Clang AST for debugging purposes.")
+                .action(ArgAction::SetTrue),
             Arg::new("emit-ir")
                 .long("emit-ir")
-                .help("Output our internal IR for debugging purposes."),
+                .help("Output our internal IR for debugging purposes.")
+                .action(ArgAction::SetTrue),
             Arg::new("emit-ir-graphviz")
                 .long("emit-ir-graphviz")
                 .help("Dump graphviz dot file.")
                 .value_name("path"),
             Arg::new("enable-cxx-namespaces")
                 .long("enable-cxx-namespaces")
-                .help("Enable support for C++ namespaces."),
+                .help("Enable support for C++ namespaces.")
+                .action(ArgAction::SetTrue),
             Arg::new("disable-name-namespacing")
                 .long("disable-name-namespacing")
                 .help(
                     "Disable namespacing via mangling, causing bindgen to \
                      generate names like \"Baz\" instead of \"foo_bar_Baz\" \
                      for an input name \"foo::bar::Baz\".",
-                ),
+                )
+                .action(ArgAction::SetTrue),
             Arg::new("disable-nested-struct-naming")
                 .long("disable-nested-struct-naming")
                 .help(
                     "Disable nested struct naming, causing bindgen to generate \
                      names like \"bar\" instead of \"foo_bar\" for a nested \
                      definition \"struct foo { struct bar { } b; };\"."
-                ),
+                )
+                .action(ArgAction::SetTrue),
             Arg::new("disable-untagged-union")
                 .long("disable-untagged-union")
                 .help(
                     "Disable support for native Rust unions.",
-                ),
+                )
+                .action(ArgAction::SetTrue),
             Arg::new("disable-header-comment")
                 .long("disable-header-comment")
                 .help("Suppress insertion of bindgen's version identifier into generated bindings.")
-                .multiple_occurrences(true),
+                .action(ArgAction::SetTrue),
```

Comment:
Ah! this is neat. It seems we were using `multiple_occurrences` incorrectly. Thank you! :heart: